app/controllers/api/v1/skills_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::SkillsController < Api::V1::BaseController
+  respond_to :json
+
+  def index
+    featured = params[:featured]
+    respond_with featured ? Skill.where(featured: featured) : Skill.all
+  end
+
```

Comment:
I think this is exactly how we have it set up with Ember right now, so let's leave it as is for now?